### ISSUE TITLE
Simulation archive interval on backward integration

### ISSUE THREAD

**Issue by hdevillepoix** [hannorein/rebound#291]
Not sure this is a bug.
`r->simulationarchive_interval` seems to need to be set negative when running a backward integration.
Otherwise it fails silently and the archive is not updated.

**Comment by hannorein** [comment]
You're probably right. I have not tried it our running a simulations backwards. I'll look into it.

**Comment by SuBok114** [comment]
I tried making the simulation archive interval negative, but received an error when trying to access the simulation later. The stats: 'SimulationgArchive' object has no attribute 'interval'.

**Comment by hannorein** [comment]
I'm not sure if this alone will do the trick. But if so, you would set the interval in the actual simulation that produces the simulation archive: `sim.simulationarchive_interval`.

**Comment by SuBok114** [comment]
The code I used that ran the actual simulation had the following line:
   sim.initSimulationArchive(safile,interval=-1.0e6)
Was this incorrect?

**Comment by hannorein** [comment]
Ah. That is the correct syntax (although I'm not sure if it will solve the issue with integrating backwards in time). Not sure why it would give you an error.

**Comment by SuBok114** [comment]
It's only when I use the next code that I get an error. The code reads:
   sa = rebound.SimulationArchive(safile)

**Comment by hannorein** [comment]
I had multiple attempts at implementing this. Every time I tried, I ended up changing almost every line of the code. The code becomes completely unreadable and the logic too complex. The cleanest way to implement this would be to have two versions of the code, one for forward and one for backwards integrations. Not nice either. 

I think the recommended solution should be a workaround on the user side: simply swap the sign of the velocities before starting the integration, then integrate forwards. 

I'll leave this issue open for now, but I won't be investing time towards a better fix at this moment.
